hat/test/org/hatdex/hat/api/service/applications/ApplicationsServiceSpec.scala

```diff
@@ -54,60 +55,46 @@ class ApplicationsServiceSpec extends ApplicationsServiceContext {
     Await.result(result, 20.seconds)
   }
 
-  it should "Include setup applications" in {
+  "The `applicationStatus` method" should "Return `active=false` status for Internal status check apps that are not setup" in {
     val service = application.injector.instanceOf[ApplicationsService]
-    val result = for {
+    val app     = Await.result(service.applicationStatus(notablesApp.id), 20.seconds).value
+    app.active mustBe false
+  }
 
+  "The `applicationStatus` parameterless method" should "Include setup applications" in {
+    val service = application.injector.instanceOf[ApplicationsService]
+    val result = for {
       _ <- service.setup(HatApplication(notablesApp, setup = false, enabled = false, active = false, None, None, None))
       apps <- service.applicationStatus()
     } yield {
       apps.length must equal(8)
       apps.find(_.application.id == notablesAppDebitless.id) must not be empty
       apps.find(_.application.id == notablesAppIncompatible.id) must not be empty
       val setupApp = apps.find(_.application.id == notablesApp.id)
-      setupApp must not be empty
-      setupApp.get.setup must equal(true)
+      setupApp.value.setup mustBe true
     }
     Await.result(result, 20.seconds)
   }
 
   "The `applicationStatus` method" should "Provide status for a specific application" in {
     val service = application.injector.instanceOf[ApplicationsService]
     val result = for {
-      app <- service.applicationStatus(notablesApp.id)
-    } yield {
-      app must not be empty
-      app.get.application.id must equal(notablesApp.id)
-    }
+      Some(app) <- service.applicationStatus(notablesApp.id)
```

Comment:
I didn't know you can do `Some(x)`. Is it used instead of a `.get`? 🤔

Reply:
the left hand side of `for` comprehension can be any pattern match (just like a `val` assignement i.e. we could also do `val Some(app) = someMethodProducingAnOption("abc")` but this particular example would not make much sense since it would throw for a `None`)